core/settings.go

```diff
@@ -33,12 +33,13 @@ type Settings struct {
 	// Deprecated: Will be removed in v0.9!
 	EmailAuth EmailAuthConfig `form:"emailAuth" json:"emailAuth"`
 
-	GoogleAuth   AuthProviderConfig `form:"googleAuth" json:"googleAuth"`
-	FacebookAuth AuthProviderConfig `form:"facebookAuth" json:"facebookAuth"`
-	GithubAuth   AuthProviderConfig `form:"githubAuth" json:"githubAuth"`
-	GitlabAuth   AuthProviderConfig `form:"gitlabAuth" json:"gitlabAuth"`
-	DiscordAuth  AuthProviderConfig `form:"discordAuth" json:"discordAuth"`
-	TwitterAuth  AuthProviderConfig `form:"twitterAuth" json:"twitterAuth"`
+	GoogleAuth      AuthProviderConfig `form:"googleAuth" json:"googleAuth"`
+	FacebookAuth    AuthProviderConfig `form:"facebookAuth" json:"facebookAuth"`
+	GithubAuth      AuthProviderConfig `form:"githubAuth" json:"githubAuth"`
+	GitlabAuth      AuthProviderConfig `form:"gitlabAuth" json:"gitlabAuth"`
+	DiscordAuth     AuthProviderConfig `form:"discordAuth" json:"discordAuth"`
+	TwitterAuth     AuthProviderConfig `form:"twitterAuth" json:"twitterAuth"`
+	MicrosoftAdAuth AuthProviderConfig `from:"microsoftAdAuth" json:"microsoftAdAuth"`
```

Comment:
I think we can remove the "Ad" suffix from the provider name and all other places, aka. let's keep it just "MicrosoftAuth".